src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http1.cs

```diff
@@ -16,7 +16,7 @@ public class HttpClientHandlerTest_Http1 : HttpClientHandlerTestBase
     {
         public HttpClientHandlerTest_Http1(ITestOutputHelper output) : base(output) { }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotNodeJS))]
```

Comment:
Why is this disabled?

Reply:
Because `node-fetch` polyfill doesn't sort headers.